es/03_5_Entendiendo_El_Descriptor.md

```diff
@@ -0,0 +1,132 @@
+# 3.5: Comprensión del descriptor
+
+> :information_source: **NOTA:** Esta sección se ha agregado recientemente al curso y es un borrador inicial que aún puede estar pendiente de revisión. Lectura con precaución.
+
+Es posible que haya notado un extraño campo `desc:` en el comando `listunspent` de la sección anterior. Esto es de qué se trata (y cómo se puede usar para transferir direcciones).
+
+> :advertencia: **ADVERTENCIA DE VERSIÓN:** Esta es una innovación de Bitcoin Core v 0.17.0 que ha continuado expandiéndose a través de Bitcoin Core 0.20.0. La mayoría de los comandos de esta sección son de 0.17.0, pero el `importmulti` actualizado que admite descriptores es de 0.18.0.
+
+## Conozca la transferencia de direcciones
```

Comment:
`## Conozca sobre transferencia de direcciones`

Reply:
Missing change. Any reason to not make it?

Reply:
- I think that `## Conozca sobre transferencia de direcciones` is better. Any reason to not change it?